### DISCUSSION TITLE
I2C Baudrate

### DISCUSSION THREAD

**Discussion by git-tomb** [PiSupply/PiJuice#718] [Q&A] [answered]
Hi,

Is there any way to make the Pijuice work with i2c baudrate set to 400000?

I'm trying to use a sense hat at that higher speed, but when I do, the Pijuice interface returns Corrupted Data.

Thanks

**Comment by tvoverbeek** [comment, marked as answer]
In /boot/config.txt:
`dtparam=i2c_arm=on,i2c_arm_baudrate=400000`

**Reply by git-tomb** [reply]
Ah,

Nevermind. Looks like it was a problem with i2c on the Pi3 B+.  Works fine on Pi4.
